roles/config-rh-sso/tasks/prereq.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: 'Pre-Req: Install required packages'
+  package:
+    name: '{{ item }}'
+    state: installed
+  with_items:
+  - firewalld
+  - python-firewall
+
+- name: 'Start firewalld service'
+  service:
+    name: firewalld
+    enabled: yes
+    state: started
+
+- name: 'Pre-Req: Open Firewall for Red Hat SSO'
+  firewalld:
+    port: '{{ item }}'
+    permanent: yes
+    state: enabled
+    immediate: yes
+  with_items:
+  - 8080/tcp
+  - 9990/tcp
+
+- name: 'Register service public IPv4'
+  os_server_facts:
+    server: at1*
```

Comment:
`at1`?

Reply:
This was part of the OpenStack specific piece that I addressed.